lib/routes/devices-and-sessions.js

```diff
@@ -21,12 +21,24 @@ module.exports = (log, db, config, customs, push, devices) => {
   // Loads and compiles a json validator for the payloads received
   // in /account/devices/notify
   const validatePushSchema = JSON.parse(fs.readFileSync(PUSH_PAYLOADS_SCHEMA_PATH))
-  const validatePushPayload = ajv.compile(validatePushSchema)
+  const validatePushPayloadAjv = ajv.compile(validatePushSchema)
   const localizeTimestamp = require('fxa-shared').l10n.localizeTimestamp({
     supportedLanguages: config.i18n.supportedLanguages,
     defaultLanguage: config.i18n.defaultLanguage
   })
 
+  function validatePushPayload(payload, endpoint) {
+    if (endpoint === 'accountVerify' && isEmpty(payload)) {
```

Comment:
This would still accept `endpoint === 'accountVerify'` with a non-empty payload, which I think we should avoid for completeness.

You *might* be able to express this in the Joi validation rules for the route, using something like:

```
  endpointAction: isA.string().valid('accountVerify').optional(),
  payload: isA.object().when('endpointAction', { is: 'accountVerify', then: isA.forbidden(), otherwise: isA.required() })
```

But I haven't tried it out.